modules/apps/object/object-service/src/main/java/com/liferay/object/internal/related/models/ObjectEntryMtoMObjectRelatedModelsProviderImpl.java

```diff
@@ -130,7 +130,8 @@ public List<ObjectEntry> getRelatedModels(
 
 	@Override
 	public int getRelatedModelsCount(
-			long groupId, long objectRelationshipId, long primaryKey)
+			boolean bypassPermission, long groupId, long objectRelationshipId,
```

Comment:
When it is M-M:getRelatedModelsCount(), you did not pass the parameter to the _objectEntryLocalService.getManyToManyObjectEntriesCount(). I would like to know why, because there are cases that getRelatedModels/getRelatedModelsCount methods are used together. So the method count should reflect the same quantity of the getRelatedModels()..

Reply:
I missed this detail. Thanks for noticing it!